Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/TabbedPage.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	using FormsElement = Forms.TabbedPage;
```

Comment:
I sense something wrong with the identation

Reply:
Look at this file: https://github.com/xamarin/Xamarin.Forms/blob/6df34004b219593885ef72e7fd2fe83b7c0fd915/Xamarin.Forms.Core/PlatformConfiguration/GTKSpecific/NavigationPage.cs

It uses spaces instead of tabs.

But most of the files use tabs instead of spaces (e.g https://github.com/xamarin/Xamarin.Forms/blob/6df34004b219593885ef72e7fd2fe83b7c0fd915/Xamarin.Forms.Core/AbsoluteLayout.cs)

My file uses TABS.

Is it your concern? Could you please explain what did you mean? Thx

Reply:
or do you hate "inside" usings? :)